Sources/Protocols/Animatable/Animatable.swift

```diff
@@ -499,6 +501,48 @@ fileprivate extension Animatable where Self: UIView {
     }
   }
 
+  func scale(fromX: Double, fromY: Double, toX: Double, toY: Double, configuration: AnimationConfiguration, completion: AnimatableCompletion? = nil) {
+    if fromX.isNaN || fromY.isNaN || toX.isNaN || toY.isNaN {
```

Comment:
I am not sure that a `guard` will be more clear here
```swift
guard !(fromX.isNaN || fromY.isNaN || toX.isNaN || toY.isNaN) else {
    return
}
``` 
```swift
guard !fromX.isNaN && !fromY.isNaN && !toX.isNaN && !toY.isNaN else {
    return
}
``` 
due to negative`!` and the code have the same complexity